src/partners/lifi.ts

```diff
@@ -66,7 +67,7 @@ const asTransfer = asObject({
 
 // Define the cleaner for the whole JSON
 const asTransfersResult = asObject({
-  transfers: asArray(asTransfer)
+  transfers: asArray(asUnknown)
```

Comment:
This is where I fixed things. We may need to re-run the partner processing on lifi, kado, and bitsofgold (so far).

Reply:
I decided to separate out this fix in its own commit.